pandas/core/indexes/base.py

```diff
@@ -4775,6 +4824,11 @@ def get_slice_bound(self, label, side, kind):
         label : object
         side : {'left', 'right'}
         kind : {'ix', 'loc', 'getitem'}
+
+        Returns
+        -------
+        int
+            Indice of label.
```

Comment:
Index (I think we're using Index everywhere, better not to introduce synonims IMO)